src/marqo/tensor_search/tensor_search.py

```diff
@@ -140,7 +144,14 @@ def create_vector_index(
     else:
         the_index_settings = configs.get_default_index_settings()
 
-    validation.validate_settings_object(settings_object=the_index_settings)
+    try:
+        validate_index_settings(
+            settings_to_validate=the_index_settings,
+            MAX_EF_CONSTRUCTION_VALUE=read_env_vars_and_defaults_ints(EnvVars.MARQO_EF_CONSTRUCTION_MAX_VALUE),
+            MAX_NUMBER_OF_REPLICAS=read_env_vars_and_defaults_ints(EnvVars.MARQO_MAX_NUMBER_OF_REPLICAS),
+        )
+    except InvalidSettingsArgError as e:
```

Comment:
We should make the code here clarify that `InvalidSettingsArgError` is from `marqo_commons` and `InvalidArgError` is from `marqo`. Or else this line seems redundant. 

We could either add a comment explaining this or import it like `from marqo_commons.shared_utils.errors import InvalidSettingsArgError as CommonsInvalidSettingsArgError`